severe-weather/generate-chips/generating_chip_data.md

```diff
@@ -15,28 +15,28 @@ conda activate severe-weather
 
 ## 2. Make Label Rasters
 
-With the conda env activated the rasters can be generated by running:
+With the `severe-weather` conda environment activated, the rasters can be generated by running:
 
 ```
 python generate_severe_weather_rasters.py
 ```
 
-This script downloads the [hwds_db](https://github.com/jrbell1/hwds_db) dataset and turns all the
-damasge swaths during Sentinel 1 into 10m rasters.
+This script downloads the [Hail and Wind Damage Swath (HWDS)](https://github.com/jrbell1/hwds_db) dataset and turns all the
+swaths during Sentinel 1 into 10m rasters.
```

Comment:
@williamh890 I'm having trouble understanding line 25 "swaths during Sentinel 1 into 10m rasters". Is there a missing verb?